### DISCUSSION TITLE
how to disable the pre-population of "press cmd+I to ask copilot to do something" text in notebook code cell

### DISCUSSION THREAD

**Discussion by bala-srm** [community/community#68081] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am using vscode insider edition and jupyter notebook. I am on macbook pro m1. When i start a new notebook or create a new code cell in  a jupyter notebook, the following text is aleady showing as a link in the new code cell - "press command +I to ask copilot to do something.Start typing to dismiss. This is very irritating. How do I disable this behaviour. I want a blank code cell and not one pre-populated with this linked message.
<img width="1512" alt="image" src="https://github.com/community/community/assets/53083124/cc918290-80a0-48da-a1ec-faeecb965c31">

**Comment by paoloose** [comment]
+1, also looking for this. so annoying.

**Comment by krippix** [comment]
I believe right-click on the message and then select disable in the context menu.

Update: I found this reply: https://github.com/microsoft/vscode/issues/237819#issuecomment-2816851582

My user preferences contain the following block now:
```json
{
    "terminal.integrated.initialHint": false,
    "chat.agent.enabled": false,
    "chat.commandCenter.enabled": false
}    
```

This seems to shut copilot up (for now)

**Comment by ananunan** [comment]
Hey! I ran into the same thing, it’s definitely a bit distracting xD
That message comes from GitHub Copilot in VS Code Insiders. It’s part of the inline help feature that shows up in empty cells to prompt you to try Copilot.
To disable it:
	1.	Open your **Command Palette** (Cmd + Shift + P)
	2.	Search for and open **“Settings (JSON)”**
	3.	Add this line to your settings:
```
"copilot.inlineChat.suggestions.showOnStartup": false
```
This should stop the message from appearing in new cells.
Let me know if that works for you \ (•◡•) /
